### PR TITLE
Update package-lock.json

### PR DESCRIPTION
**Pull request by mathiasbynens**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,30 +1,28 @@
 {
   "name": "jsperf.com",
   "version": "2.0.0",
-  "description": "jsPerf aims to provide an easy way to create and share test cases, comparing the performance of different JavaScript snippets by running benchmarks",
+  "description":
+    "jsPerf aims to provide an easy way to create and share test cases, comparing the performance of different JavaScript snippets by running benchmarks",
```
Comment (maxbeatty) [inline package.json]: prettier may have nabbed this :/

File: package.json [hunk 2]
```diff
@@ -94,6 +92,28 @@
   },
   "snyk": true,
   "now": {
-    "alias": "jsperf.com"
+    "alias": "jsperf.com",
+    "env": {
```
Comment (maxbeatty) [inline package.json]: thought this was easier to manage

**Comment by maxbeatty** [comment]
Hijacking this branch to debug [this](https://github.com/jsperf/jsperf.com/pull/451#issuecomment-351176982) 😈 will you try to deploy this branch to now? should have more insight into why it isn't starting up with these changes

File: package.json (hunk 2 above)
Comment (mathiasbynens) [reply package.json]: It totally is! Thanks

**Comment by mathiasbynens** [comment]
Deploying…

**Comment by mathiasbynens** [comment]
```
> ▲ npm start
> > jsperf.com@2.0.0 start /home/nowuser/src
> > node server.js
> 171212/210547.789, [log,debug] data: registering auth strategies
> 171212/210549.577, [log,info,db] data: executed 0 migrations
> 171212/210549.598, [log,info] data: connecting to redis cache
> No open port found. Make sure 'npm start' creates an HTTP server on a port (any).
```

File: server/lib/cache.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ exports.register = function (server, options, next) {
     server.expose('set', client.set.bind(client));
     server.expose('drop', client.drop.bind(client));
 
-    client.start(next);
+    client.start().then(next).catch(next);
```
Comment (maxbeatty) [inline server/lib/cache.js]: catbox client start changed to being promise-based after v7 (oops) hopefully this solves the stall when connecting to rediscover

**Comment by mathiasbynens** [comment]
That fixes it! ♥

**Pull request merged**